packages/ag-charts-community/src/scene/shape/linearGradientFill.ts

```diff
@@ -12,40 +11,58 @@ export class LinearGradientFill extends Shape {
     @SceneChangeDetection({ redraw: RedrawType.MAJOR })
     stops?: string[] = undefined;
 
+    get mask(): Path | undefined {
+        return this._mask;
+    }
+    set mask(newMask: Path | undefined) {
+        if (this._mask != null) {
+            this.removeChild(this._mask);
+        }
+
+        if (newMask != null) {
+            this.appendChild(newMask);
```

Comment:
Am I right in thinking that because we override the `render` method, and the `group`'s render method is responsible for rendering children, we won't draw this mask?